src/pages/Auth/SignupPage/SignupPage.module.scss

```
@@ -0,0 +1,331 @@
+@use "src/utils/scss/media";
+
+.form-input {
+  width: 100%;
+  height: 45px;
+  box-sizing: border-box;
+  border: #d2dae2 1px solid;
+  font-size: 15px;
+  padding-left: 20px;
+  margin-bottom: 12px;
+
+  &::placeholder {
+    font-size: 15px;
+    letter-spacing: -0.8px;
+    color: #bec9d5;
+  }
+
+  &:disabled {
+    background-color: #e5eaf0;
+    border: 1px solid #d2dae2;
+    color: #b5c1cd;
+  }
+
+  &--invalid {
+    border: 2px solid red;
+  }
+
+  &--disabled-value:disabled {
+    background-color: #175c8e;
+    color: #fff;
+  }
+
+  @include media.media-breakpoint(mobile) {
+    width: 320px;
+    height: 36px;
+    padding-left: 16px;
+
+    &::placeholder {
+      font-size: 14px;
+    }
+  }
+
+  &--half {
+    width: 192px;
+
+    @include media.media-breakpoint(mobile) {
+      width: 155px;
+    }
+  }
+
+  &--flex-end {
+    order: 99;
+  }
+}
+
+.select {
+  position: relative;
+  display: inline-block;
+  margin-left: 6px;
+
+  &--flex-end {
+    order: 99;
+  }
+
+  &__trigger {
+    position: relative;
+    width: 192px;
+    height: 45px;
+    padding: 0 15px;
+    box-sizing: border-box;
+    background-color: #fff;
+    font-size: 15px;
+    font-weight: 400;
+    line-height: 1.2;
+    letter-spacing: -0.8px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border: 1px solid #d2dae2;
+
+    @include media.media-breakpoint(mobile) {
+      width: 155px;
+      font-size: 14px;
+      height: 36px;
+    }
+
+    &::after {
+      position: absolute;
+      top: 11px;
+      right: 15px;
+      background: no-repeat center/17px 12px url("https://static.koreatech.in/assets/img/bus_dropdown.png");
+      width: 17px;
+      height: 17px;
+      content: "";
+    }
+
+    &--active {
+      background-color: #175c8e;
+      color: #fff;
+    }
+  }
+
+  &__content {
+    position: absolute;
+    z-index: 5;
+  }
+
+  &__option {
+    width: 192px;
+    height: 45px;
+    padding: 0 15px;
+    box-sizing: border-box;
+    background-color: #fff;
+    cursor: pointer;
+    font-size: 15px;
+    font-weight: 400;
+    line-height: 45px;
+    letter-spacing: -0.8px;
+    color: #252525;
+    border: 1px solid #d2dae2;
+
+    &:hover {
+      background-color: #175c8e;
+      color: #fff;
+    }
+
+    @include media.media-breakpoint(mobile) {
+      width: 155px;
+    }
+  }
+}
+
+.signup {
+  display: flex;
+  flex-wrap: wrap;
+
```

반응형에서 디자인이 전부 깨집니다.
속성이 추가로 들어가야 할 것 같네요.

```suggestion
.signup {
  display: flex;
  flex-wrap: wrap;
  @include media.media-breakpoint(mobile) {
    justify-content: center;
  }
```

---

[498b27c](https://github.com/BCSDLab/KOIN_WEB_RECODE/pull/14/commits/498b27c0504d09ac9309b1bfbcf3dc6d7b720506) 수정하였습니다.